themes/bootstrap3/js/channels.js

```diff
@@ -59,6 +59,16 @@ VuFind.register('channels', function Channels() {
     }
     record.data('bs.popover').options.content = html;
   }
+
+  // Truncate lines to height with ellipses
+  function clampLines(el) {
+    let words = el.innerHTML.split(" ");
+    while(el.scrollHeight > el.offsetHeight) {
+      words.pop();
+      el.innerHTML = words.join(" ") + "...";
```

Comment:
I think it should be an ellipsis character (U+2026), not three dots.

Reply:
I checked using "…" this with several fonts didn't see any notable difference in the way the ellipses were displayed or the number of words that were truncated. I'm not against the idea though!

Reply:
I'd support using … -- if nothing else, it's one character instead of three. :-)